plugins/memblaze/memblaze-nvme.c

```diff
@@ -402,11 +414,12 @@ int parse_params(char *str, int number, ...)
 	return 0;
 }
 
-static int mb_get_additional_smart_log(int argc, char **argv, struct command *cmd, struct plugin *plugin)
+static int mb_get_additional_smart_log(
+	int argc, char **argv,
+	struct command *cmd, struct plugin *plugin)
 {
 	struct nvme_memblaze_smart_log smart_log;
-	char *desc =
-	    "Get Memblaze vendor specific additional smart log (optionally, for the specified namespace), and show it.";
+	char *desc = "Get Memblaze additional smart log, and show it.";
```

Comment:
Overlong strings are okay to be one line. This is an exception but if the shortened help text is also fine. Either way you like it.